spec/models/user_spec.rb

```diff
@@ -325,4 +325,49 @@
       end
     end
   end
+
+  describe '#age' do
+    let(:user) { create(:user, first_name: 'John', last_name: 'Doe') }
+    before do
+      mocked_time = Time.parse('08/12/2020')
+      allow(Time).to receive(:now).and_return(mocked_time)
+    end
+
+    context 'with no birthdate' do
+      before { user.birthdate = nil }
+      it { expect(user.age).to eq nil }
+    end
+
+    context 'with birthdate' do
+      context 'in a leap year and the same month and day' do
+        before { user.birthdate = '08/12/2000' }
+        it { expect(user.age).to eq 20 }
+      end
+
+      context 'in a leap year one day before birthday' do
+        before { user.birthdate = '07/12/2000' }
```

Comment:
We should always use ISO dates

Reply:
True! I used this one as if it was a setting instead a field. My bad